### DISCUSSION TITLE
Is there any way to check the available flash space in tasmota?

### DISCUSSION THREAD

**Discussion by ayykamp** [arendst/Tasmota#10497] [Q&A] [answered]
So I bought some random tuya bulbs and flashed tasmota on the with tuya-convert, is there any way to find out how big the flash is?

**Comment by meingraham** [comment, marked as answer]
`Status 4` from the Console or the Information page on the web UI.

**Comment by sfromis** [comment]
You can still upgrade Tasmota via compressed binaries, from the OTA server (no download first) even when it appears that the "available" flash space is not enough. Tasmota does a two-step upgrade via the smaller minimal binary, and the compressed .gz binaries also allows a larger download

In fact, upgrade to full current Tasmota is recommended after Tuya-convert, instead of the older cut-down version.
